### PR TITLE
Setup the skeleton for the CXL plugin

### PR DESCRIPTION
**Pull request by umeshghediya**
Setup Basic skeleton for the WP CLI command

### CODE REVIEW

File: Commands/class-cxl-cli.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * CLI commands for CXL
+ */
+class CLX_CLI extends WP_CLI_Command {
```
Comment (lkraav) [inline Commands/class-cxl-cli.php:5]: CXL, not CLX.

File: Commands/class-cxl-cli.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?php
```
Comment (lkraav) [inline Commands/class-cxl-cli.php:1]: Everything must be namespaced.

File: cxl-wp-cli-commands.php [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+require_once __DIR__ . '/helpers.php';
+
+/**
+ * Autoloader for cli commands class.
+ *
+ * @param string $class Class name to load file.
+ *
+ * @return void
+ */
+function cxl_autoloader( $class ) {
```
Comment (lkraav) [inline cxl-wp-cli-commands.php:27]: Let's use composer autoloader.

File: cxl-wp-cli-commands.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Plugin Name:       CXL WP_CLI Commands
+ * Description:       WP CLI commands for export subscription order to external API.
+ * Version:           1.0.0
```
Comment (lkraav) [inline cxl-wp-cli-commands.php:5]: 0.1.0 makes more sense to start.

**Review by lkraav — CHANGES_REQUESTED** [review]

File: Commands/class-cxl-cli.php (hunk 2 above)
Comment (umeshghediya) [reply Commands/class-cxl-cli.php:1]: Done

File: cxl-wp-cli-commands.php (hunk 4 above)
Comment (umeshghediya) [reply cxl-wp-cli-commands.php]: Done

File: cxl-wp-cli-commands.php (hunk 3 above)
Comment (umeshghediya) [reply cxl-wp-cli-commands.php:24]: Okay sure, but can we keep this, for now, once cleared with other PRs we'll change this one, as currently needs to change in 7 more PRs.

File: cxl-wp-cli-commands.php [hunk 5]
```diff
@@ -10,13 +10,10 @@
  * @package cxl
  */
 
-if ( ! class_exists( 'WP_CLI' ) ) {
```
Comment (lkraav) [inline cxl-wp-cli-commands.php]: What is this line change about?

Comment (umeshghediya) [reply cxl-wp-cli-commands.php]: Space was added by mistake.

**Pull request merged**